NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -87,23 +92,45 @@ class ProtectedAPIViewController: UIViewController {
         accountResult?.signOut()
 
         accountResult = nil
-        
-        accessToken = nil
+        accessTokenAPI1 = nil
+        accessTokenAPI2 = nil
 
         showResultText("Signed out")
-
         updateUI()
     }
     
-    @IBAction func protectedAPIPressed(_: Any) {
-        guard let accessToken = self.accessToken else {
-            let errorMessage = "No access token found"
-            print(errorMessage)
-            showResultText(errorMessage)
+    @IBAction func protectedApi1Pressed(_: Any) {
+        guard   let url = protectedAPIUrl1,
+                !protectedAPIScopes1.isEmpty else {
```

Comment:
My understanding is that `protectedAPIScopes1.isEmpty` will not be true even when API 1 is not configured, because we assign a non-empty value to this array as default value

Reply:
We assign just a "template" of a value. If a user tries to run the app with this template, the app will return an error. So, users will have to enter a real value or empty the array.

Reply:
Ok, but this check isn't useful because `protectedAPIScopes1` will never be empty because the default value is a non-empty array.

Reply:
Updated, now it works as expected